### ISSUE TITLE
Angular workspace with projects

### ISSUE THREAD

**Issue by kbjerke** [CanopyTax/single-spa-angular#66]
Hi, I was looking into creating a monorepo type project with multiple angular applications, and I decided to go for a project structure workspace, supported by default using the angular cli.

```
# Create workspace without default application
ng new project-name --createApplication=false
cd project-name
# Generate application, will by default output to /projects/{appName}
ng generate application app1
ng generate application app2
# Add single-spa-angular to configuration for both projects
ng add single-spa-angular@beta --project app1
ng add single-spa-angular@beta --project app2
# Make sure they are both valid single-spa apps
ng build app1 && ng build app2
```
This will crash, because the builder is unable to locate src/main.single-spa.ts, and in this case the main should be projects/{appName}/src/main.single-spa.ts.

https://github.com/CanopyTax/single-spa-angular/blob/4d396e2961a082b5dc703d64feeeb1ec1d51b9e0/src/schematics/ng-add/index.ts#L74

Also I found an issue with what I'm guessing is this line:

https://github.com/CanopyTax/single-spa-angular/blob/4d396e2961a082b5dc703d64feeeb1ec1d51b9e0/src/builders/single-spa-webpack-builder.ts#L7

Which causes all apps to be exported with the same module name, and replacing the global object when including them e.g.: if app1 is default project:
```
(function webpackUniversalModuleDefinition(root, factory) {
	if(typeof exports === 'object' && typeof module === 'object')
		module.exports = factory();
	else if(typeof define === 'function' && define.amd)
		define([], factory);
	else if(typeof exports === 'object')
		exports["app1"] = factory();
	else
		root["app1"] = factory();
})
```
It will actually crash if during registerApplication() as it cannot find the window["appName"].default object.

By manually editing defaultProject to empty string in angular.json:
`"defaultProject": ""`
You get the expected result.

**Comment by joeldenning** [comment]
Fixed with #67

**Comment by kbjerke** [comment]
@joeldenning , I started working on a fix for the case where:
https://github.com/CanopyTax/single-spa-angular/blob/4d396e2961a082b5dc703d64feeeb1ec1d51b9e0/src/schematics/ng-add/index.ts#L74
Would solve the fact that every application generated in a workspace points to src/main.single-spa.ts. Unfortunately I didn't have time to finish. But I figured it would be something like: 
```
clientProject.architect['single-spa'].options.main = `${project.workspace.sourcePath}/src/main.single-spa.ts`;
```

And altering the test case to:
```
  test('should modify angular.json', () => {
    const tree = testRunner.runSchematic<NgAddOptions>('ng-add', { routing: true, project: 'ss-angular-cli-app' }, defaultAppTree);
    ...
    expect(ssApp.architect['single-spa'].options.main).toBe('src/main.single-spa.ts');
    ...
  });
```

And the tests passed, but the cli was acting strange. Steps to reproduce would be:
```
# create new workspace with default application
ng new project
cd project
# add single-spa-angular to default project, seems to work
ng g single-spa-angular:ng-add
# generate new application in project (non-default)
ng generate application app1
ng g single-spa-angular:ng-add --project app1
# This executes successfully, but points to the wrong main.single-spa.ts file, if you delete /projects/app1/src/main.single-spa.ts and re-run: ng g single-spa-angular:ng-add --project app1, it would correct the entry in angular.json to be correct.
```
